src/content/docs/infrastructure/host-integrations/host-integrations-list/nuxt-js-integration.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Nuxt.js integration
+tags:
+  - nuxt.js integration
+  - nuxt integration
+  - new relic integrations
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to query your New Relic data with NRQL, our SQL-like query language."
+redirects:
+---
+import nuxtjsdashboard from 'images/infrastructure_screenshot-crop_nuxtjs-dashboard.png'
+
+With our Nuxt.js dashboard, you can easily track your application error rate, initial page load average, route changes, throughput, and page views with JavaScript errors. Built with our infrastructure agent and our Vue browser monitoring, the Nuxt.js dashboard lets you view critical information all in one place.
+
+<img
+  title="Nuxt.js dashboard"
+  alt="A screenshot of a Nuxt.js dashboard"
+  src={nuxtjsdashboard}
+/>
+
+## 1. Install infrastructure agent [#infra]
+Before you can get your data into New Relic, you should first install the infrastructure agent (if you have it installed, you can skip this step). This enables New Relic to collect data so that application error rates, throughput, and page views with JavaScript errors can appear in dashboards. This step is optional, but we strongly recommend you install the infrastructure agent to see all your host and browser monitoring data.
+
+There are two ways to install the infrastructure agent:
+* Follow the [guided install](https://one.newrelic.com/marketplace?state=607786be-99a9-fa66-4b67-3f7f3b9551d4) to instrument our system with the infrastructure agent.
+* [Install the infrastructure agent](docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) using the command line. 
+
+
+## 2. Install Vue browser monitoring [#install]
+After installing the infrastructure agent, the next step is to install the browser monitor. From **[one.newrelic.com](https://one.newrelic.com)**, click **Add data > Browser & mobile > Vue**.
+Follow the instructions from there to generate the JavaScript code snippet used in the next step.
+
+## 3. Configure the integration [#config]
+To add the JavaScript snippet in the Nuxt.js application:
+1. Create a JavaScript file in the `static` folder:
+  ```sql
+  touch <FILENAME>.js
+  ```
+2. Create a function in the newly created JavaScript file to execute the generated script:
+  ```sql
+  nano <FILENAME>.js
+  ```
+  ```sql
+  function <FUNCTION_NAME>() {
+	<GENERATED SCRIPT IS PASTED HERE BY REMOVING OPENING AND CLOSING <script> TAGS>
+  }
+  FUNCTION_NAME();
+  ```
+3. Add the created JavaScript to the nuxt configuration file named `nuxt.config.js`:
```

Comment:
Add the created JavaScript file to the nuxt configuration file named `nuxt.config.js`: